parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -1231,33 +1237,9 @@ impl_runtime_apis! {
 	}
 }
 
-struct CheckInherents;
-
-impl cumulus_pallet_parachain_system::CheckInherents<Block> for CheckInherents {
-	fn check_inherents(
```

Comment:
@slumber Unfortunately I don't think we can safely expect all downstream users to enact this same change for the moment.

Reply:
We should restore the `CheckInherents` logic to `register_validate_block!` but not use it in any runtimes that have the `aura_ext::FixedVelocityConsensusHook`.

@BradleyOlson64 this is something for the update guide.